lib/windows/app/actions/deviceActions.js

```diff
@@ -156,8 +156,19 @@ function deviceSetupInputReceivedAction(input) {
     };
 }
 
+const NORDIC_VENDOR_ID = 0x1915;
+const NORDIC_BOOTLOADER_PRODUCT_ID = 0x521f;
+
 function logDeviceListerError(error) {
     if (error.usb) {
+        // It is expected to get libusb errors for bootloader, skipping them:
```

Comment:
If I was reading this source code, I would like to see more information in this comment (among other things, because the error is not expected in my linux environment).

In my experience, it's good for this kind of hacks to have a lengthier explanation, so that people reading this code can understand *why* the hack was put there in the first place.

Something along the lines of: 

> In win32 platforms, a USB device with no interfaces which are bound to a libusb-compatible-driver
> will fail to enumerate and trigger a LIBUSB_* error. This is the case of a nRF52840 in DFU mode.
> We want to *not* show an error to the user in this particular case.